Dockerfile.centos

```diff
@@ -0,0 +1,56 @@
+FROM centos:7 AS build-env
+LABEL maintainer "Sida Chen <sidchen@redhat.com>"
+
+ARG GIT_TAG
+RUN test -n "$GIT_TAG" # Expect Clair GIT TAG to be non-empty.
+
+# Install Golang by retrieving the binary
+ENV GO_VERSION=1.11.5
+ENV GO_OS=linux
+ENV GO_ARC=amd64
+RUN curl https://dl.google.com/go/go${GO_VERSION}.${GO_OS}-${GO_ARC}.tar.gz --output go.tar.gz
+RUN echo ff54aafedff961eb94792487e827515da683d61a5f9482f668008832631e5d25 go.tar.gz > GOCHECKSUM
+RUN sha256sum -c GOCHECKSUM
+RUN tar -C /usr/local -xzf go.tar.gz > /dev/null
+ENV GOPATH=/go
+ENV PATH=$PATH:/usr/local/go/bin:${GOPATH}/bin
+
+# Verify go verion
+RUN go version
+
+# Install dependencies
+RUN yum install -y git bzr rpm xz gcc
+# musl-dev 
+RUN go get -u github.com/cloudflare/cfssl/cmd/cfssl
+RUN go get -u github.com/cloudflare/cfssl/cmd/cfssljson
+RUN go get -u github.com/coreos/jwtproxy/cmd/jwtproxy
+
+# Build Clair with git tag
+RUN go get -u -d github.com/coreos/clair/cmd/clair
+WORKDIR /go/src/github.com/coreos/clair
+RUN git checkout $GIT_TAG
+RUN go install github.com/coreos/clair/cmd/clair
+
+FROM centos:7 AS runtime-env
+LABEL maintainer "Sida Chen <sidchen@redhat.com>"
+
+# Install init system and Clair depended binaries
+RUN yum install -y python-setuptools git bzr rpm xz
```

Comment:
No need for bzr, Ubuntu uses git now.
We also need to be very careful that this is rpm4 and not rpm5. They are two totally different things made by different people.